Libraries/LibWeb/Animations/Animatable.cpp

```diff
@@ -140,7 +140,7 @@ void Animatable::add_transitioned_properties(Vector<Vector<CSS::PropertyID>> pro
         auto delay = delays[i]->is_time() ? delays[i]->as_time().time().to_milliseconds() : 0;
         auto duration = durations[i]->is_time() ? durations[i]->as_time().time().to_milliseconds() : 0;
         auto timing_function = timing_functions[i]->is_easing() ? timing_functions[i]->as_easing().function() : CSS::EasingStyleValue::CubicBezier::ease();
-        VERIFY(timing_functions[i]->is_easing());
+        //VERIFY(timing_functions[i]->is_easing());
```

Comment:
How has this anything to do with the next changes? We shouldn't just disable something, especially if it's a `VERIFY` - those usually point to bugs if they trigger.

Reply:
I'll investigate further, could be a conflict with a recent commit

Reply:
This seems like it should be fixed by #4449, so rebasing on `master` could be the solution here.